src/platform/qt/GameController.cpp

```diff
@@ -39,41 +39,11 @@ using namespace std;
 
 GameController::GameController(QObject* parent)
 	: QObject(parent)
-	, m_drawContext(nullptr)
-	, m_frontBuffer(nullptr)
-	, m_threadContext()
-	, m_activeKeys(0)
-	, m_inactiveKeys(0)
-	, m_logLevels(0)
-	, m_gameOpen(false)
-	, m_vf(nullptr)
-	, m_useBios(false)
-	, m_override(nullptr)
 	, m_audioProcessor(AudioProcessor::create())
-	, m_pauseAfterFrame(false)
-	, m_sync(true)
 	, m_videoSync(VIDEO_SYNC)
 	, m_audioSync(AUDIO_SYNC)
-	, m_fpsTarget(-1)
-	, m_turbo(false)
-	, m_turboForced(false)
-	, m_turboSpeed(-1)
-	, m_wasPaused(false)
-	, m_audioChannels()
-	, m_videoLayers()
-	, m_autofire{}
-	, m_autofireStatus{}
-	, m_stateSlot(1)
-	, m_backupLoadState(nullptr)
-	, m_backupSaveState(nullptr)
 	, m_saveStateFlags(SAVESTATE_SCREENSHOT | SAVESTATE_SAVEDATA | SAVESTATE_CHEATS | SAVESTATE_RTC)
 	, m_loadStateFlags(SAVESTATE_SCREENSHOT | SAVESTATE_RTC)
```

Comment:
These would require including the serialization header in the header file and wasn't sure if you would want that (more files need to recompile if a change is made in the serialization header), rather than one cpp file. This was also my reasoning for the ones below as well. Do you still want me to move them there anyway?